python/test_dcr_template.py

```diff
@@ -20,19 +20,29 @@
 #
 
 from __future__ import print_function, division, absolute_import
+import copy
 import numpy as np
 
 import lsst.afw.geom as afwGeom
 from lsst.afw.geom import Angle
 import lsst.afw.image as afwImage
 import lsst.afw.math as afwMath
 import lsst.meas.algorithms as measAlg
-from lsst.sims.photUtils import PhotometricParameters
 import unittest
 import lsst.utils.tests
 from .dcr_template import DcrModel
 from .dcr_template import DcrCorrection
+from .dcr_template import solve_model
 from .dcr_template import wrap_warpExposure
+from .dcr_template import calculate_rotation_angle
+from .dcr_template import parallactic_angle
+
+
+nanFloat = float("nan")
+nanAngle = Angle(nanFloat)
+lsst_lat = Angle(np.radians(-30.244639))
```

Comment:
This code will be switched to the LSST layout and use EUPS in DM-9614. 
I've updated the test code documentation.